vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -368,7 +385,6 @@ def _is_wNa16_group_channel(self, weight_quant: BaseModel,
     def _get_scheme_from_parts(
             self, weight_quant: BaseModel,
             input_quant: BaseModel) -> "CompressedTensorsScheme":
```

Comment:
Can you specify "int" in all the places you have w4a8? In the near future we will have float_wXaY so I'd like to avoid conflicts